Morphic/Morphic.pier

```diff
@@ -323,21 +330,21 @@ all crossMorphs answer ==true== to the ==handlesMouseDown:== message.
 
 @@todo Add this method to ==CrossMorph==:
 
-[[[caption=Declaring that ==CrossMorph== will react to mouse clicks.
-CrossMorph>>>handlesMouseDown: anEvent
-	^true
+[[[language=smalltalk|caption=Declaring that ==CrossMorph== will react to mouse clicks|label=scr:mouseEvent
+CrossMorph>>#handlesMouseDown: anEvent
```

Comment:
Yeah, much of the book uses `>>>` in the method definition scripts. Pillar compiles it to a non-ascii character »
In any case, I'm merging the changes, we can sort out which notation we prefer, separately.